docs/elasticsearch_connector.rst

```diff
@@ -27,42 +27,96 @@ Quickstart
 In this Quickstart, we use the Elasticsearch connector to export data produced by the Avro console
 producer to Elasticsearch.
 
-Start Zookeeper, Kafka and Schema Registry if you haven't done so. You also need to have
-Elasticsearch running locally or remotely and make sure that you know the address to connect to
-Elasticsearch.
+First, start all the necessary services using Confluent CLI:
 
-.. ifconfig:: platform_docs
+.. tip::
 
-   The instructions on how to start these services are available at the
-   :ref:`Confluent Platform quickstart<quickstart>`.
+   If not already in your PATH, add Confluent's ``bin`` directory by running: ``export PATH=<path-to-confluent>/bin:$PATH``
+
+.. sourcecode:: bash
+
+   $ confluent start
+
+Every service will start in order, printing a message with its status:
+
+.. sourcecode:: bash
+
+    Starting zookeeper
+    zookeeper is [UP]
+    Starting kafka
+    kafka is [UP]
+    Starting schema-registry
+    schema-registry is [UP]
+    Starting kafka-rest
+    kafka-rest is [UP]
+    Starting connect
+    connect is [UP]
```

Comment:
We can get rid of the paragraph right after this since we told them how to start the services.